tests/h/streamer/messages_test.py

```diff
@@ -12,13 +12,11 @@
 
 class FakeSocket:
     client_id = None
-    filter = None
```

Comment:
This gets added by `SocketFilter` and the precise name is private to `SocketFilter` now. We can change it without having to modify these tests again.